CoreArrangements/src/au/gov/asd/tac/constellation/arrangements/spectral/GraphSpectrumEmbedder.java

```diff
@@ -62,8 +62,8 @@
 
     private static enum MatrixType {
 
-        AdjacencyMatrix,
-        LaplacianMatrix,;
+        ADJACENCYMATRIX,
```

Comment:
With multiple words, can we separate them with an underscore so it would read better? So it would be `ADJACENCY_MATRIX` and `LAPLACIAN_MATRIX`?

Reply:
The changes have been made apologies for the delay.